CRM/Utils/Check/Component/Env.php

```diff
@@ -594,8 +594,8 @@ public function checkExtensions() {
       $messages[] = new CRM_Utils_Check_Message(
         __FUNCTION__,
         $e->getMessage(),
-        ts('Extension download error'),
-        \Psr\Log\LogLevel::ERROR,
+        ts('Unable to connect to public extension directory'),
```

Comment:
Nitpick: could it be "Unable to connect to the extension directory"?

Having "public" hints that there is a private directory too. And the feed might be configured to connect to a private/third-party directory anyway.